client/src/pages/CreateOrganisationProfile.js

```diff
@@ -147,6 +147,11 @@ const CreateOrgProfile = (props) => {
             type: CREATE_Organisation_ERROR,
             error: `Creating organisation failed, reason: ${message}`,
           });
+          setError(
+            "location",
+            "required",
+            "Please select an address from the drop-down",
```

Comment:
(according to #1204 ) they want the message to be `Address is required` instead, so it is consistent with other fields, e.g. Email is required.